src/OpenTelemetry.Instrumentation.Runtime/RuntimeMetrics.cs

```diff
@@ -165,7 +165,7 @@ static RuntimeMetrics()
             description: "The amount of time the JIT compiler has spent compiling methods since the process start.");
 #endif
 
-#if NETCOREAPP3_1_OR_GREATER
+#if NET6_0_OR_GREATER
```

Comment:
There are multiple instances of `#if NET6_0_OR_GREATER` used in code now. For example, the `#endif` above also corresponds to the same condition. These can be simplified.

Reply:
This `#if NET6_0_OR_GREATER` also places the UpDownCounter metric: `"process.runtime.dotnet.assemblies.count"` outside of the `#if NET6_0_OR_GREATER` condition.

Reply:
yes. assembly count API is available in older fw too,so no need to conditional compile.